core/services/log/orm.go

```diff
@@ -70,9 +91,36 @@ func (o *orm) FindConsumedLogs(fromBlockNum int64, toBlockNum int64) ([]LogBroad
 	return broadcasts, err
 }
 
+func (o *orm) CreateBroadcast(blockHash common.Hash, blockNumber uint64, logIndex uint, jobID int32, qopts ...postgres.QOpt) error {
+	q := postgres.NewQ(o.db, qopts...)
+	res, err := q.Exec(`
+        INSERT INTO log_broadcasts (block_hash, block_number, log_index, job_id, created_at, updated_at, consumed, evm_chain_id)
+		VALUES ($1, $2, $3, $4, NOW(), NOW(), false, $5)
+    `, blockHash, blockNumber, logIndex, jobID, o.evmChainID)
+	if err != nil {
+		return errors.Wrap(err, "while creating log broadcast")
+	}
+	if err != nil {
+		return errors.Wrap(err, "while creating log broadcast")
+	}
+	rowsAffected, err := res.RowsAffected()
+	if err != nil {
+		return errors.Wrap(err, "CreateBroadcast failed")
+	}
+	if rowsAffected == 0 {
+		return errors.Wrap(err, "while marking log broadcast as consumed")
```

Comment:
error message should mention rowsAffected likely

Reply:
Updated to return `sql.ErrNoRows`, which appears to be the most common for this pattern.